flow/project.py

```diff
@@ -3645,8 +3911,9 @@ def _main_status(self, args):
                 error = error.__cause__  # Always show the user traceback cause.
             traceback.print_exception(type(error), error, error.__traceback__)
         else:
+            length_jobs = len(self) if aggregates is None else len(aggregates)
```

Comment:
This is incorrect if any aggregates are defined and `aggregates is None` isn't it? `None` implies all jobs and aggregates correct?